### PR TITLE
Added Processing IDE

### PR DESCRIPTION
**Pull request by msurguy**
# Please make sure all below checkboxes have been checked before submitting your PR.

- [x] I have read and understood the [contribution guidelines](https://github.com/thibmaek/awesome-raspberrypi/blob/master/CONTRIBUTING.md).
- [x] This pull request has a descriptive title. *(For example: `Add Raspbian`)*
- The topic I added
	- [x] includes a valid (https) link,
	- [x] includes a concise and on-topic description,
	- [x] mentions if there is only support some devices,
	- [x] is not a duplicate,
	- [x] has been added at the bottom of the appropriate category,
	- [x] is in the form of **Named Link - Description, separated by commas.**
	- [x] ends with a dot.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,8 +73,7 @@ The complete collection of Raspberry Pi models consist of:
 - [FabScanPi](https://mariolukas.github.io/FabScanPi-Server/) - FabScanPi is an open source 3D laser scanner using the Raspberry Pi Camera Module. ![Supports Raspberry Pi 2+](/media/badges/rpi-2+.png)
 - [Rocket Show](https://rocketshow.net) - Play synced backing tracks, videos and DMX light-sequences live on stage. ![Supports Raspberry Pi 3](/media/badges/rpi-3.png)
 - [SamplerBox](http://www.samplerbox.org/makeitsoftware) - Drop'n'play sampler: drop .WAV samples on the SD card, and play!
-
-
+- [Processing](https://github.com/processing/processing/releases/download/processing-0264-3.3.7/processing-3.3.7-linux-raspbian.zip) - A pre-installed and out-of-the-box version of Processing Development Environment.
```
Comment (thibmaek) [inline README.md]: Please do not include direct downloads in this list. Remove this line completely and link to the project page in tools instead

File: README.md [hunk 2]
```diff
@@ -143,12 +142,13 @@ The complete collection of Raspberry Pi models consist of:
 - [Stratux](https://github.com/cyoung/stratux) - Open source ADS-B reciever which feeds weather, traffic, GPS, and AHRS data to electronic flight bag software via wifi.
 - [Voice Kit](https://aiyprojects.withgoogle.com/voice) - AIY Voice Kit from Google, to build a standalone voice recognition system using the Google Assistant, or add voice recognition and natural language processing to your Raspberry Pi-based projects.
 - [speed-camera](https://github.com/pageauc/speed-camera) - Object Motion Tracking uses python, openCV, USB Cam or picamera module to record speed data.
-- [pi-timolo](https://github.com/pageauc/pi-timolo) - Remote Headless multi feature PiCamera Operation from Rclone Remote Storage Service and More. 
+- [pi-timolo](https://github.com/pageauc/pi-timolo) - Remote Headless multi feature PiCamera Operation from Rclone Remote Storage Service and More.
```
Comment (thibmaek) [inline README.md]: Please do not modify other people's commits. Can you undo this change?

File: README.md [hunk 3]
```diff
@@ -143,12 +142,13 @@ The complete collection of Raspberry Pi models consist of:
...
 - [Hearing aid prototoype](https://github.com/m-r-s/hearingaid-prototype) - A Raspberry Pi powered prototype of a hearing aid. ![Supports Raspberry Pi 3](/media/badges/rpi-3.png)
 - [RPI tempmon](https://github.com/gavinlyonsrepo/raspberrypi_tempmon) - CPU GPU temperature monitor with various functions such as LED GPIO, Graph output, email, alarm limit, notifications and logging.
 - [ RPi Motor Library](https://github.com/gavinlyonsrepo/RpiMotorLib) - Python 3 library to connect various motors & servos to the Pi.
 - [Circle](https://github.com/rsta2/circle) - A C++ bare metal environment for Raspberry Pi.
 - [Project MyHouse](https://maxoffsky.com/research-progress/project-myhouse-a-smart-dollhouse-with-gesture-recognition/) - Smart Doll House with Gesture Recognition, using Raspberry Pi 3 or Pi Zero and PSMove motion controller.
+- [Processing](https://pi.processing.org/get-started/) - Processing Development Environment running on Raspberry Pi.
```
Comment (thibmaek) [inline README.md]: This is OK, should be the only line this PR adds.

File: README.md (hunk 2 above)
Comment (msurguy) [reply README.md]: @thibmaek I tried to undo this but my text editors automatically remove this space no matter how hard I try to keep it, I am kinda stuck

**Comment by thibmaek** [comment]
✨ Thanks @msurguy fixed that space inline myself
